core/src/test/java/com/linecorp/armeria/common/DefaultRequestHeadersBuilderTest.java

```diff
@@ -142,6 +148,120 @@ void schemeFromSessionProtocol() {
                 .isInstanceOf(IllegalArgumentException.class);
     }
 
+    @Test
+    void testAcceptLanguageMultiHeader() {
+        final RequestHeaders headers = RequestHeaders
+                .builder()
+                .path("/")
+                .method(HttpMethod.GET)
+                .add(HttpHeaderNames.ACCEPT_LANGUAGE,
+                        "de-us ;q=0.9, *;q=0.8",
+                        "en;q=0.95, en-US;q=0.98"
+                )
+                .build();
+        final List<LanguageRange> acceptLanguages = headers.acceptLanguages();
+        assertEquals(
+                ImmutableList.of(
+                        new LanguageRange("en-US", 0.98),
+                        new LanguageRange("en", 0.95),
+                        new LanguageRange("de-us", 0.9),
+                        new LanguageRange("*", 0.8)
+                ),
+                acceptLanguages
+        );
```

Comment:
Global comment: Could you use `assertThat` of Assert4J? We prefer Assert4J when asserting in tests.
https://armeria.dev/community/developer-guide#use-assert4j-instead-of-junits-assertion-api

Reply:
Oops. Fixed.